docs/Queries/Filters.md

```diff
@@ -449,8 +452,8 @@ For more examples, see [[#Due Date]].
 
 - `no start date`
 - `has start date`
-- `starts (before|after|on) <date>`
-- `starts (before|after|in) <date range>`
+- `starts (before|on or before|after|on or after|on) <date>`
```

Comment:
picking another random location, we haven't talked about this before - and it's not yet in the Contributing Guide, but please could you update the 'What's New'-type section in Introduction.md? Thanks.

Reply:
sure, done =)
hope my wording is ok there.